src/main/webapp/styles/main.css

```diff
@@ -21,14 +21,30 @@ td {
   border : 1px solid gray;
   vertical-align: top;
 }
-.price {
+.price, .quantity {
   text-align: right;
 }
 img.product-tile {
   max-width: 64px;
 }
 
+.error  {
+  color: red;
+}
+
+.success {
+  color: green;
+}
+
+.recentlyViewedProduct {
```

Comment:
Why vertical display of recently viewed, do you like it?) Just hint, no changes required